realm/realm-library/src/main/java/io/realm/BaseRealm.java

```diff
@@ -85,7 +85,7 @@ public void onSchemaVersionChanged(long currentVersion) {
                                 RealmCache.updateSchemaCache((Realm) BaseRealm.this);
                             }
                         }, true);
-        this.schema = new RealmSchema(this);
+        this.schema = new StandardRealmSchema(this);
```

Comment:
Shouldn't this be delegated to subclasses for construction? Since right now a DynamicRealm would also get this schema?